src/borg/archiver.py

```diff
@@ -2733,10 +2733,16 @@ def main():  # pragma: no cover
             tb = "%s\n%s" % (traceback.format_exc(), sysinfo())
             exit_code = e.exit_code
         except RemoteRepository.RPCError as e:
-            msg = "%s %s" % (e.remote_type, e.name)
-            important = e.remote_type not in ('LockTimeout', )
+            important = e.exception_class not in ('LockTimeout', )
             tb_log_level = logging.ERROR if important else logging.DEBUG
-            tb = sysinfo()
+            if important:
+                msg = e.exception_full
+            else:
+                msg = e.get_message()
+            tb = ""
+            for line in e.sysinfo.strip(('\n')).split('\n'):
```

Comment:
the parens seem to indicate that you had a tuple of characters here, but with only 1 element and no comma, it is no tuple any more. also .splitlines() maybe?